### DISCUSSION TITLE
posts/tools-i-use-2024-mid/

### DISCUSSION THREAD

**Discussion by giscus** [jerrylususu/jerrylususu.github.io#34] [Blog Comments]
# posts/tools-i-use-2024-mid/

在 MacOS 下重建工作流；顺带还提及了各类工具中使用的插件

https://nekonull.me/posts/tools-i-use-2024-mid/

**Comment by lians0** [comment]
Obsidian 的git插件会遇到提交的时候会卡 和文件数据有关

**Reply by jerrylususu** [reply]
可能和数据量以及文件类型相关；我的使用场景基本是纯markdown+少量图片，目前没有遇到过卡顿

**Comment by wuqi** [comment]
浏览器可以用Vivaldi  ,可以装chrome插件,而且可以直接记笔记,还可以新建会话保存会话,新建工作区,还有鼠标手势,超方便,不需要加很多插件原生就有很多功能

**Reply by jerrylususu** [reply]
感谢分享！其实 edge 我主要离不开的是 devtools 里可以选中请求然后直接编辑并重放（edit and resend）的功能；目前似乎只有 firefox 有类似的，其他浏览器还没有原生支持（虽然 edge 里这个功能也常常是某个版本好，下个版本就又坏了） :joy:

**Reply by wuqi** [reply]
有个简单办法,copy里有个copy as fetch ,然后贴到控制台,也能用...
